src/traces/parcoords/attributes.js

```diff
@@ -12,10 +12,12 @@ var colorAttributes = require('../../components/colorscale/color_attributes');
 var colorbarAttrs = require('../../components/colorbar/attributes');
 var colorscales = require('../../components/colorscale/scales');
 var axesAttrs = require('../../plots/cartesian/layout_attributes');
+var fontAttrs = require('../../plots/font_attributes');
 
 var extendDeep = require('../../lib/extend').extendDeep;
 var extendFlat = require('../../lib/extend').extendFlat;
 
+
```

Comment:
Now that there aren't giant outstanding PRs, might be time to revisit: https://github.com/plotly/plotly.js/pull/1371